### ISSUE TITLE
Strange Brigade (312670)

### ISSUE THREAD

**Issue by Oliver-1xok** [ValveSoftware/Proton#920]
The game [Strange Brigade](https://store.steampowered.com/app/312670/Strange_Brigade/) don't works on my system. It installs without problems. But after starting only a small black and non-movable window with unsteady border appears.

My System:
```
CPU: i7 6700k
GPU: GTX 970 
RAM: 16 GB
NVIDIA Driver Version: 396.54
Proton Version: proton-3.7beta-20180824
OS: Xubuntu 18.4. (Ubuntu with Xfce)
```
Maybe it would help like with Doom, if I enter a launch parameter, which starts the game directly with Vulkan. But I don't know such a launch parameter for this game.

**Comment by AlexNigl** [comment]
You can click slightly below the top left corner to start the game, where I drew the green box.
![strange_brigade](https://user-images.githubusercontent.com/2684552/47102132-2d056880-d23c-11e8-8c28-155c96b427fa.png)

I have no audio but this is probably a separate issue.

**Comment by GloriousEggroll** [comment]
note: this still doesnt work with faudio, but installing xact to the prefix and using the xaudio2_7 override works

**Comment by flibitijibibo** [comment]
The latest FAudio deals with one issue where the submix ordering was wrong...

https://github.com/FNA-XNA/FAudio/commit/8e0883d9b3e3d0dedde83a234eb93e551b9b4a4f

... but I've narrowed down the hideous noise to one of the submix effects. If you comment out this block, the sound is okay:

https://github.com/FNA-XNA/FAudio/blob/master/src/FAudio_internal.c#L1005

I have no idea what the problem effect in question is, but if it's an effect in the engine and not one of ours, we could be in trouble...

**Comment by flibitijibibo** [comment]
Well, this isn't good. Out of the 7 unique effects I was able to identify, _none_ of them are known effects. They're not ours and the CLSIDs are not familiar to me at all.

It seems like we're doing everything right; all the flags are accounted for, LockForProcess succeeds, and the effect in question is in-place so there shouldn't be any uninitialized memory (the bug sounds the same every time, FWIW). The only thing I can possibly think of is parameter update accuracy, but without access to the effects I have no idea if that's relevant.

Point being, it's somewhere in either SetEffectChain (FAudio.c) or ProcessEffectChain (FAudio_internal.c) but that's all I know.

**Comment by flibitijibibo** [comment]
Out of random curiosity I decided to add support for the SILENT flag in the XAPO spec and... uh, it sounds okay now?

https://github.com/FNA-XNA/FAudio/commit/5f8e1f0bc71867dca5310143bc3ceecc778023fd
EDIT: [Did I really think f32 PCM would never be < 0...](https://github.com/FNA-XNA/FAudio/commit/10ee03e10d5abc0aca8661a098b481cb6f746c1d#diff-a66dcf666075459e26eb20e93062ace1R640)

I don't feel like this should make sense, but I did notice that they don't set their effect parameters until 1 update after it starts processing, so maybe it's a very specific case where silence initializes some stuff and by the time real sound starts pushing out, it coincidentally happens to be ready for real samples. I'm _completely_ fabricating that idea though, so if you try upstream FAudio be on the lookout for more screeching.

**Comment by flibitijibibo** [comment]
3.16-6 includes the latest FAudio, should resolve sound issues for this title.

**Comment by gort818** [comment]
Strange brigade runs great now using the vulkan exe. getting an avg of 90fps on vega 56 ultra @1440p. Audio works perfectly now thanks!

**Comment by darksithk8** [comment]
Hello everyone.
I have similar issue with running game Strange Brigade on Ubuntu system.
When game loads then goes back to desktop.

I have fresh install of Ubuntu:
Description: Ubuntu 18.04.1 Bionic LTS
I have everything up to date.
Using Proton 3.16-6 Beta.

This is my PC hardware [Lenovo Y720]:

                         system         80VR (LENOVO_MT_80VR_BU_idea_FM_Y720-15
/0                        bus            LNVNB161216
/0/0                      memory         128KiB BIOS
/0/4                      processor      Intel(R) Core(TM) i5-7300HQ CPU @ 2.50G
/0/24/0                   memory         8GiB SODIMM DDR4 Synchronous Unbuffered
/0/100                    bridge         Intel Corporation
/0/100/1                  bridge         Skylake PCIe Controller (x16)
/0/100/1/0                display        GP106M [GeForce GTX 1060 Mobile]
/0/100/2                  display        Intel Corporation
/0/100/14                 bus            Sunrise Point-H USB 3.0 xHCI Controller
/0/100/14/0    usb1       bus            xHCI Host Controller
/0/100/14/0/8             generic        Xbox Embedded Wireless
/0/100/14/0/9             input          ITE Device(8910)
/0/100/14/0/b             communication  Bluetooth wireless interface
/0/100/14/1    usb2       bus            xHCI Host Controller
/0/100/14.2               generic        Sunrise Point-H Thermal subsystem
/0/100/15                 generic        Sunrise Point-H Serial IO I2C Controlle
/0/100/15.2               generic        Intel Corporation
/0/100/16                 communication  Sunrise Point-H CSME HECI #1
/0/100/17                 storage        Sunrise Point-H SATA Controller [AHCI m
/0/100/1c                 bridge         Sunrise Point-H PCI Express Root Port #
/0/100/1c/0    wlp2s0     network        Wireless 8265 / 8275
/0/100/1c.3               bridge         Sunrise Point-H PCI Express Root Port #
/0/100/1c.3/0  enp3s0     network        RTL8111/8168/8411 PCI Express Gigabit E
/0/100/1c.4               bridge         Sunrise Point-H PCI Express Root Port #
/0/100/1d                 bridge         Sunrise Point-H PCI Express Root Port #
/0/100/1d/0               storage        Non-Volatile memory controller
/0/100/1f                 bridge         Sunrise Point-H LPC Controller
/0/100/1f.2               memory         Memory controller
/0/100/1f.3               multimedia     Intel Corporation
/0/100/1f.4               bus            Sunrise Point-H SMBus
![sb github com valvesoftware proton issues 920b](https://user-images.githubusercontent.com/37878553/52397592-5d586780-2ab6-11e9-8d92-e24116942de3.png)

**Comment by kisak-valve** [comment]
Hello @darksithk8, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by darksithk8** [comment]
It's a lot of code:
[steam-312670.log](https://github.com/ValveSoftware/Proton/files/2841076/steam-312670.log)

**Comment by darksithk8** [comment]
Anyone?

**Comment by NTMan** [comment]
The game mostly playable if not enabled tessellation. With enabled tessellation even impossible select character because of graphics artifacts. I don't know which render are used by default (DX12 or Vulkan) so I can't say which are culprit here vkd3d or RADV driver.
https://youtu.be/3qv2jGMWtlU

**Comment by kisak-valve** [comment]
Hello @NTMan, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by NTMan** [comment]
> Hello @NTMan, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

[steam-312670.log](https://github.com/ValveSoftware/Proton/files/3104432/steam-312670.log)
[System Information](https://gist.github.com/NTMan/33cca4906328793206218192820779ec)
[Strange Brigade benchmark with enabled tessellation](https://youtu.be/ByLojdARu6Q)

**Comment by darksithk8** [comment]
@NTMan, but Mr. kisak-valve does not give solution to problem only collecting logs.. :(

**Comment by hakzsam** [comment]
The tessellation issue with RADV is fixed with:
https://cgit.freedesktop.org/mesa/mesa/commit/?id=b155f74d7bfc9daec7794074c4f644025fbddf8b

The fix should be in the next 19.0 release (19.0.4) and it will be in 19.1 of course.

**Comment by ryliejamesthomas** [comment]
Audio-wise I'm getting nothing but noise. Is this what happened in earlier error reports?

edit: More research suggests this is actually an issue in Windows too, and disabling audio reverb via the launcher fixes it.

**Comment by Oschowa** [comment]
The launcher crashes again on latest Proton experimental 6.3-20210625. Proton 6.3-5 is fine.
[steam-312670.log](https://github.com/ValveSoftware/Proton/files/6726944/steam-312670.log)

**Comment by kisak-valve** [comment]
Hello @Oschowa, your log looks similar to https://github.com/ValveSoftware/Proton/issues/980#issuecomment-869145030.

**Comment by rbernon** [comment]
The regression should be fixed in today's Experimental update, both games were affected.

**Comment by Oschowa** [comment]
Can confirm it's working again, thanks!

**Comment by aufkrawall** [comment]
Getting only a black screen with Nvidia on Plasma Wayland/XWayland with latest Proton Experimental/9.0-4. Not sure if it has ever worked before. Likely Nvidia-specific issue and maybe not even related to Proton itself.

**Comment by ThePawlow** [comment]
Getting only a black screen even on an AMD GPU. Perhaps its a Wayland issue? My friend is winning an argument. Please help me.
I need this game to work so I can smugly prove that gaming on Linux is totally viable

**Hardware Specifications**

[System Information](https://gist.githubusercontent.com/ThePawlow/fd93eb8fd411999f0da243f35dd3149e/raw/cca7159e29a11e99ebc351912b7704b2f16a001c/system_information.txt)
CPU: Intel Core i9-14900K (32 logical processors screaming into the void)
GPU: AMD Radeon RX 7900 XTX (24GB VRAM of unfulfilled potential)
RAM: 32GB (none of which Strange Brigade is using because it won't launch)
Motherboard: Gigabyte Z790 AERO G (I don't think that's the problem - Right?)

**My Journey of Suffering (Testing Environment)**
In my desperate quest to prove he's wrong, I've tested this across with a fresh install:

Bazzite (current - used in Logs) - Nope
Zorin OS - Nada
Ubuntu 25.10 - Nice try
Ubuntu 24.04 LTS - No LTS for Strange Brigade* Yet?

**Proton Versions Tested**
Every single Proton version that was mentioned in this conversation was tested. All of them. I have no dignity left.

**Logs**
[steam-312670-proton-6.3-8g-dx12.log](https://github.com/user-attachments/files/24683306/steam-312670-proton-6.3-8g-dx12.log)
[steam-312670-proton-6.3-8g-vulkan.log](https://github.com/user-attachments/files/24683301/steam-312670-proton-6.3-8g-vulkan.log)
[steam-312670-proton-9.0-4f-dx12.log](https://github.com/user-attachments/files/24683304/steam-312670-proton-9.0-4f-dx12.log)
[steam-312670-proton-9.0-4f-vulkan.log](https://github.com/user-attachments/files/24683303/steam-312670-proton-9.0-4f-vulkan.log)
[steam-312670-proton-10.0-3-dx12.log](https://github.com/user-attachments/files/24683305/steam-312670-proton-10.0-3-dx12.log)
[steam-312670-proton-10.0-3-vulkan.log](https://github.com/user-attachments/files/24683302/steam-312670-proton-10.0-3-vulkan.log)

**Additional**
My friend's "I told you so" grows stronger with each passing day. Time is of the essence.

P.S. - If you fix this, you're not just fixing a proton version. You're settling a bet. You're defending honor. You're proving that a grown adult making the switch to Linux wasn't my first terrible life choice in 2026

_No pressure though._

**Comment by simifor** [comment]
@ThePawlow It's working fine on my end, and was also working fine on august 2025. Currently: 9060 XT, 7800X3D, mesa 25.3.1, linux 6.18.5, Proton 10 and experimental with vulkan and dx12.

From your logs it looks like you're loading from an external partition, what format is it?

**Comment by ThePawlow** [comment]
Hey @simifor !
Very odd, are you using Wayland?
It's ext4, I just saw that for some reason the partition is called Data (Game Dev). Resulting in an odd mounting point called
> /run/media/pawlow/Data\ \(Games_Dev\)/
```Disk /dev/nvme1n1: 1.82 TiB, 2000398934016 bytes, 3907029168 sectors
Disk model: Samsung SSD 9100 PRO 2TB                
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
```

**Comment by simifor** [comment]
@ThePawlow yes, I use plasma wayland. Something you can try to avoid some mounting path woes is setting this as your launch parameter on the game's steam properties window: `PROTON_SET_GAME_DRIVE=1 %command%`

**Comment by ThePawlow** [comment]
I've now installed the game along my OS Partition. Still black screen.
> /var/home/pawlow/.local/share/Steam/steamapps/common/StrangeBrigade/

EDIT:
Booting with the properties did not fix it neither
> PROTON_SET_GAME_DRIVE=1 %command%

EDIT 2:
I wanted to testify, that I run all games from my external disk without issues! 😓 

Additionally I wanted to share this picture, the launcher looks different than the one above. So something might changed in the launchers resources too?

Looks like a horribly handled WPF/ WinForms Application 😂 

<img width="640" height="820" alt="Image" src="https://github.com/user-attachments/assets/afe691d6-f0cb-4360-9be4-b172540be9ea" />

**Comment by simifor** [comment]
@ThePawlow the launcher looks the same on my end, haven't checked how it looks on windows.
As for things to try

[GraphicsOptions.txt](https://github.com/user-attachments/files/24697919/GraphicsOptions.txt)
While there aren't many options, it's worth a try replacing your existing graphics options (rename .txt to .ini).
Path is `.../SteamLibrary/steamapps/compatdata/312670/pfx/drive_c/users/steamuser/AppData/Local/Strange Brigade/`

**Comment by ThePawlow** [comment]
Replaced with your config - Did not work, sadly

**Comment by simifor** [comment]
@ThePawlow as you think wayland may be causing issues on your end, have you tried a different desktop environment? I think bazzite omes with SDDM so you should be able to easily switch between desktop environments in the login screen. You can also try installing gamescope and trunning the game with `gamescope -- %command%`

**Comment by ThePawlow** [comment]
Hey once again @simifor !
> try installing gamescope and trunning the game

It now starts the Launcher in an extra Window with the same graphic artifacts. The game itself just flickers.
> have you tried a different desktop environment

I've tried KDE (Brazzite) and GNOME (Ubuntu). With the latest Brazzite Version I cannot change from Wayland back to X11.
I will try CachyOS X11 next - Slowly loosing my mind 😂

EDIT:
Tried latest CachyOS 6.18.6 with X11 and Wayland. Running DX12, Vulkan with and without Gamescope - Same problem!
Now I'm out of ideas.

**Comment by runar-work** [comment]
I tested this on Wayland with RX 7600 on Bazzite (Proton Experimental and Mesa 25.3.3) and Arch Linux (various Mesa and Proton versions), but the only thing I came across was an issue I had with the Vulkan renderer specifically which I've filed [here](https://gitlab.freedesktop.org/mesa/mesa/-/issues/14705).

**Comment by kisak-valve** [comment]
1+ Hour's Loading Stage on Ubuntu 24.04 with Intel CPU

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9521.
@wanlizhu posted on 2026-02-23T19:34:21:

# Compatibility Report
- Name of the game with compatibility issues: Strange Brigade
- Steam AppID of the game: 312670

## System Information
- CPU: intel i9 14900K 
- GPU: NVIDIA RTX 5060 Ti
- Video driver version: nvidia 590.48.01
- Kernel version: 6.14.0
- Link to full system information report (and system diagnostics) as [Gist](https://gist.github.com/wanlizhu/a7fe1924c49af25156eb10e73a3f5ea1):
- Proton version: Proton Hotfix and GE-Proton10-29

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [ ] that I have checked whether there are updates for my system available.

[steam-312670.log](https://github.com/user-attachments/files/25498445/steam-312670.log)

## Symptoms 
Strange Brigade includes an in-game benchmark mode, but when I start the benchmark the display remains black for about an hour before the benchmark actually begins. This hour-long “loading” period is unexpected and suggests something abnormally slow is occurring during initialization.
* Loading time with Proton Hotfix: 1+ hour 
* Loading time with GE-Proton10-29: about 30 minutes 

I profiled the black-screen phase using a perf flamegraph and monitored the system with btop, and saw no meaningful CPU utilization, disk I/O, or network activity. The game appears to be doing essentially nothing beyond waiting on wineserver; at the same time, wineserver itself remains idle and does not show any long-running work. Notably, I can only reproduce this issue  on Intel CPUs, and there is no such issue on AMD CPUs.

## Reproduction
Launch Strange Brigade on Ubuntu 24.04 with intel i9 14900K, and then you'll see a prolonged black screen.
